tools/mo/openvino/tools/mo/convert.py

```diff
@@ -262,6 +271,12 @@ def convert_model(
         :param stream_output:
             Switch model conversion progress display to a multiline mode.
 
+    PaddlePaddle-specific parameters:
+        :param example_input:
```

Comment:
We already have 'example_input' for PyTorch. Can we move 'example_input' to a common group of parameters?

Reply:
> We already have 'example_input' for PyTorch. Can we move 'example_input' to a common group of parameters? To "Framework-agnostic" or "Optional" group.

Maybe not. The `example_input` is limited to `inline conversion`. If it is moved to "Framework-agnostic" or "Optional" group, it will be exposed in MO command. User may never pass the parameter to MO, which is unexpected.